src/generate/handlebarsConfig.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { getValue } from "./utils";
```

Comment:
`getValue` is being used as a helper, so I think it should be exported from the helpers file so that it doesn't need special handling.